jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcFlowRepository.java

```diff
@@ -269,18 +344,24 @@ public Flow update(Flow flow, Flow previous) throws ConstraintViolationException
     }
 
     @SneakyThrows
-    private Flow save(Flow flow, CrudEventType crudEventType) throws ConstraintViolationException {
+    private FlowWithSource save(Flow flow, CrudEventType crudEventType, String flowSource) throws ConstraintViolationException {
         // validate the flow
         modelValidator
             .isValid(flow)
             .ifPresent(s -> {
                 throw s;
             });
 
+        try {
+            flowSource = flowSource != null ? flowSource : JacksonMapper.ofYaml().writeValueAsString(flow);
```

Comment:
Does a flowSource with null is possible? 
I don't think so, throw an exception if null